### PR TITLE
Translate readme.md into Traditional Chinese (readme-zh_tw.md)

### PR DESCRIPTION
**Pull request by RivelleDays**
## Checklist:

- [X] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

The readme.md file has been translated into Traditional Chinese. The translation uses concise, professional, and neutral language, tailored to the preferences of Traditional Chinese users, particularly those in Taiwan.

### CODE REVIEW

File: .github/readme.md
Comment (Cohee1207) [inline .github/readme.md:1]: Why did you delete the original?

**Review by Cohee1207 — CHANGES_REQUESTED** [review]

File: .github/readme.md
Comment (RivelleDays) [reply .github/readme.md:1]: Sorry, I don’t have experience in this area. Should I keep the original version? Let me check… I initially thought it would be fine to upload a new version.
